src/content/App/App.types.ts

```diff
@@ -4,28 +4,21 @@ export enum SectionType {
   lab,
 }
 
-// TODO refactor this
 export enum Term {
-  summerOne,
-  summerTwo,
-  summerFull,
-  winterOne,
-  winterTwo,
-  winterFull,
+  One,
+  Two,
 }
 
+export const ALL_TERMS_SET = new Set([Term.One, Term.Two])
+
 export enum Views {
   calendar,
   settings,
 }
 // TODO refactor this
 export const Term_String_Map: { [key in Term]: string } = {
```

Comment:
Do we really need this? Can we just move it to the enum

Reply:
I added it to reduce duplication but looks like I only ended up needing it one place — removed.